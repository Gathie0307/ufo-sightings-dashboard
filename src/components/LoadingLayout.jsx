const LoadingLayout = () => {
  return (
    <div className='flex flex-col animate-pulse'>
      <div className='mx-auto mr-2 bg-gray-200 bg-border-2 rounded-md border-gray-200 w-[800px] h-[382px]'></div>
      <div className='flex justify-between items-center mt-8'>
        <div className='mt-4 rounded-full px-4 py-2 bg-gray-200 text-white w-[150px] h-[42px]'></div>
        <div className='mt-4 rounded-full px-4 py-2 bg-gray-200 text-white w-[150px] h-[42px]'></div>
      </div>
    </div>
  );
};

export default LoadingLayout;
