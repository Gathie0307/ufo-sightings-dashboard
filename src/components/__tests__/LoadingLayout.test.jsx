import { render } from '@testing-library/react';
import LoadingLayout from '../LoadingLayout';

describe('Loading Layout', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<LoadingLayout />);

    expect(asFragment()).toMatchSnapshot();
  });
});
