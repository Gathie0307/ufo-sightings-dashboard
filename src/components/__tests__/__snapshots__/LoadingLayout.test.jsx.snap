// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loading Layout > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col animate-pulse"
  >
    <div
      class="mx-auto mr-2 bg-gray-200 bg-border-2 rounded-md border-gray-200 w-[800px] h-[382px]"
    />
    <div
      class="flex justify-between items-center mt-8"
    >
      <div
        class="mt-4 rounded-full px-4 py-2 bg-gray-200 text-white w-[150px] h-[42px]"
      />
      <div
        class="mt-4 rounded-full px-4 py-2 bg-gray-200 text-white w-[150px] h-[42px]"
      />
    </div>
  </div>
</DocumentFragment>
`;
